indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -332,6 +333,22 @@ def _process(self):
         return send_file('registrations.csv', csv_file, 'text/csv')
 
 
+class RHRegistrationsPrintBadges(RHRegistrationsActionBase):
+    """Print badges for the selected registrations"""
+
+    def _process(self):
+        badge_templates = self.event.getBadgeTemplateManager().getTemplates().items()
+        badge_templates.sort(key=lambda x: x[1].getName())
+        pdf_options = WConfModifBadgePDFOptions(self.event).getHTML()
+        badge_design_url = url_for('event_mgmt.confModifTools-badgePrinting', self.event)
+        create_pdf_url = url_for('event_mgmt.confModifTools-badgePrintingPDF', self.event)
+
+        return WPManageRegistration.render_template('management/print_badges.html', self.event, regform=self.regform,
+                                                    templates=badge_templates, pdf_options=pdf_options,
+                                                    registrations=self.registrations, registration_ids=self.ids,
```

Comment:
I think it's nicer to use known-to-be-clean data over data coming straight from the client